aeron-tools/src/main/java/uk/co/real_logic/aeron/tools/MediaDriverOptions.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright 2015 Kaazing Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.co.real_logic.aeron.tools;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.commons.cli.CommandLine;
```

Comment:
In the future, we'll be sure to run them by you for anything we're planning on doing a PR for; how about for now?  Can we agree on using Apache's CommandLine now?  I don't think there's a license incompatibility.  The alternative is for us to write and include our own getopt parsing class, which seems a little silly.